src/test/java/com/epam/hw2/ExerciseFirst.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.hw2;
+
+import org.openqa.selenium.*;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+
+public class ExerciseFirst extends AbstractWebPageTest {
+
+    @Test(groups = {"group-hw2"}, priority = 1)
+    public void exerciseFirstTest() {
+        webDriver().manage().window().maximize();
+        webDriver().manage().timeouts()
+                .implicitlyWait(2, TimeUnit.SECONDS);
+        // 1 | Open test site by URL
+        webDriver().navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+        // 2 | Assert Browser title
+        asert().assertEquals(webDriver().getTitle(), "Home Page");
+        // 3 | Perform login username: Roman  pass: Jdi1234
+        webDriver().findElement(By.cssSelector("nav .uui-profile-menu a")).click();
+
+        webDriver().findElement(By.id("name")).sendKeys("Roman");
+        webDriver().findElement(By.id("password")).sendKeys("Jdi1234");
+        webDriver().findElement(By.id("login-button")).click();
+        // 4 | Assert Username is loggined "ROMAN IOVLEV" Name is displayed
+        asert().assertEquals("ROMAN IOVLEV", webDriver()
+                                                    .findElement(By.id("user-name")).getText());
+        // 5 | Assert that there are 4 items on the header section are displayed
+        //!"HOME", "CONTACT FORM", "SERVICE", "METALS & !COLORS"
+        List<WebElement> topMenu = webDriver()
+                      .findElements(By.cssSelector("nav .uui-navigation.nav.navbar-nav.m-l8>li>a"));
+        ArrayList menuPart01 = new ArrayList<>();
```

Comment:
Please parameterize Lists
```suggestion
        ArrayList<String> menuPart01 = new ArrayList<>();
```

\+ in other places too